enterprise_subsidy/apps/api/v1/views/subsidy.py

```diff
@@ -326,12 +325,14 @@ def create(self, request, *args, **kwargs):
     )
     def destroy(self, request, *args, **kwargs):
         """
-        Delete a subsidy
+        Soft delete a subsidy by setting its is_soft_deleted field to True
 
         Endpoint Location: DELETE /api/v1/subsidies/{uuid}/
         """
-        response = super().destroy(request, kwargs['uuid'])
-        return response
+        instance = self.get_object()
+        instance.is_soft_deleted = True
+        instance.save()
```

Comment:
It'd be better to override the `delete()` method on the model to do this, so that any other consumer of the model (the Django Admin, management commands, or whatever) will have almost no choice but to soft-delete.
Doing a soft-delete inside the model definition would also let you remove the admin modifications around delete permissions in this PR.